Modules/Module_Processing/Module_Population/xASL_adm_DeleteManyTempFiles.m

```diff
@@ -46,8 +46,8 @@ function xASL_adm_DeleteManyTempFiles(x)
     %% 2. Native space — ASL module files
     fprintf('Deleting temporary ASL files in native space folders:    ');
 
-    for iP=1:length(Files2DelNativeASL)
-        xASL_TrackProgress(iP, length(Files2DelNativeASL));
+    for iPath=1:length(Files2DelNativeASL)
+        xASL_TrackProgress(iPath, length(Files2DelNativeASL));
         
         for iSession=1:length(listASLDone)
             sessionIndex = regexp(listASLDone{iSession}, 'ASL_\d*$');
```

Comment:
iP -> iPath on line 58!!!

Reply:
Thanks! Done in comment "review suggestions Jan"